package.json

```diff
@@ -23,20 +23,30 @@
     "localenv:compose:psql": "docker compose -f ./localenv/cloud-nine-wallet/docker-compose.yml -f ./localenv/happy-life-bank/docker-compose.yml ${BEFORE_MERGED:-} -f ./localenv/merged/docker-compose.yml",
     "localenv:compose": "BEFORE_MERGED='-f ./localenv/tigerbeetle/docker-compose.yml' pnpm localenv:compose:psql",
     "localenv:seed:auth": "pnpm -C ./packages/auth knex seed:run --env=development && pnpm -C ./packages/auth knex seed:run --env=peerdevelopment",
-    "sanity": "pnpm -r build && pnpm -r test"
+    "sanity": "pnpm -r build && pnpm -r test",
+    "localenv:autopeer:start": "run-p tunnel:start wait-tunnel:localenv:compose",
+    "localenv:autopeer:start:psql": "run-p tunnel:start wait-tunnel:localenv:compose:psql",
```

Comment:
2 comments:
* should we call it `localenv:compose:autopeer` and `localenv:compose:psql:autopeer`?
* I think we would ideally only start cloud-nine-wallet. But that can be a follow up issue/PR